README.md

```diff
@@ -21,6 +21,11 @@ Please refer to [Manual Setup Build Environment](docs/HowToSetEnv.md).
 
 For more information of libraries in the TizenRT Docker Image, see https://hub.docker.com/r/tizenrt/tizenrt/.
 
+Especially For Windows environment, configuring CRLF is required as below.
```

Comment:
As I left a comment before, let's move this next of `2. Getting TizenRT source code`. Because this should be executed on forked folder.